apps/server/src/modules/authentication/controller/account.controller.ts

```diff
@@ -22,4 +22,14 @@ export class AccountController {
 	): Promise<void> {
 		await this.accountUc.changePasswordForUser(currentUser.userId, userId, password);
 	}
+
+	@Patch('me')
+	async updateMyAccount(@CurrentUser() currentUser: ICurrentUser, @Body() params: PatchMyAccountParams): Promise<void> {
+		return this.accountUc.updateMyAccount(currentUser.userId, params);
+	}
+
+	@Put('me/password')
```

Comment:
Hmm. We have chosen PATCH for 'updateMyAccount' since that is a partial update never a create (part of a resource as you said). We have chosen PUT for 'updateMyPassword' since this is creating or updating the entire sub-resource (the use case is that no password exists, just a temporary password, hence 'my real password' is created or updated here). I'll rename the controllers method to 'setMyPassword' to have the naming match better.

Reply:
As discussed, I'll make everything a 'Patch' operation.